movies/2.context.md

```diff
@@ -0,0 +1,102 @@
+# The JSON-LD context
+
+## Story
+
+As an API provider, I want to describe the semantics in a JSON-LD context, in order to simplify the actual JSON responses.
+
+## API behaviour
+
+```http
+GET https://hydra-movies.herokuapp.com/
+```
+
+Response body (excerpt):
+
+```json
+{
+  "@context": "/context.jsonld",
```

Comment:
I realise that relative context URL is a side-effects of running a static file server.

```suggestion
  "@context": "https://hydra-movies.herokuapp.com/context.jsonld",
```

In real APIs however I would encourage absolute links so that the representation is self-contained without the need to apply a `baseIRI` when deserialising. Not to mention that figuring out the correct base can get hairy if an API does `303` redirects. Can lead to easy mistakes on the client end.